src/python/pants/backend/python/tasks/resolve_requirements_task_base.py

```diff
@@ -33,16 +33,13 @@ def prepare(cls, options, round_manager):
     round_manager.require_data(PythonInterpreter)
     round_manager.optional_product(PythonRequirementLibrary)  # For local dists.
 
-  def resolve_requirements(self, req_libs, local_dist_targets=None):
+  def resolve_requirements(self, req_libs):
     """Requirements resolution for PEX files.
 
     :param req_libs: A list of :class:`PythonRequirementLibrary` targets to resolve.
-    :param local_dist_targets: A list of :class:`PythonDistribution` targets to resolve.
     :returns: a PEX containing target requirements and any specified python dist targets.
     """
     tgts = req_libs
```

Comment:
I think we could kill this local var and use req_libs directly in `self.invalidated`.

Reply:
D'oh! Of course. Good eye. Fixed.